compressor/compressor.py

```
@@ -1,60 +1,86 @@
+    '''
+        Scheduled job to compress data when number of 
+        entries reaches a threshold.
+    '''
     count = getNumberOfEntries()
+    print("Number of Entries:", count)
 
-    if count > 3:
-        entries = getFirstNEntries()
-        print(entries)
+    if count >= COMPRESS_THRESHOLD:
+        entries = getFirstNEntries(COMPRESS_THRESHOLD)
```

Ruff (0.8.2)

Local variable `entries` is assigned to but never used

Remove assignment to unused variable `entries`

(F841)